### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by louisehendrickx** [vmikk/PhyloNext#14]
It would be nice to get an overall value for PD in the results considering all data (without spatial component). The reason we want to do this, is to understand phylogenetic diversity loss when we submit multiple different phylogenies.

**Comment by shawnlaffan** [comment]
Do you mean PD for all taxa in a sample?  If so then that's pretty simple to calculate.  

BTW, you might also be interested in some of the per-branch indices for PD clade contribution and potential loss.
https://github.com/shawnlaffan/biodiverse/wiki/Indices#pd-clade-contributions
https://github.com/shawnlaffan/biodiverse/wiki/Indices#pd-clade-loss
https://github.com/shawnlaffan/biodiverse/wiki/Indices#pd-clade-loss-ancestral-component

**Comment by louisehendrickx** [comment]
Yes, indeed: PD for all taxa in a sample. How should it be calculated then?
